src/bindgen.odin

```diff
@@ -1549,18 +1553,25 @@ main :: proc() {
 	context.temp_allocator = permanent_allocator*/
 
 	ensure(len(os.args) == 2, "Usage: bindgen directory")
-	input_directory := os.args[1]
+	input_arg := os.args[1]
 
-	if !os.is_dir(input_directory) {
-		fmt.panicf("%v is not a directory", input_directory)
+	config_filename := "bindgen.sjson"
+	config_dir: string
+	if strings.has_suffix(input_arg, ".sjson") && os.is_file(input_arg) {
```

Comment:
We could remove the suffix check here and allow for any file name/ext to be used, and always try to load as `sjson`? But enforcing the file type is also nice. Figured I would note it either way!